.changeset/old-lobsters-pull.md

```diff
@@ -0,0 +1,7 @@
+---
+"firebase": minor
+"@firebase/firestore-types": minor
+"@firebase/firestore": minor
+---
+
+Release Firestore Bundles as a prototype patched feature.
```

Comment:
You should describe in your Release Notes what this is ("bundles" is not a term that developers will understand - at least not yet) and how they can use it (mention the additional import). The Prototype Patching is an implementation detail that doesn't need to go into the release notes. We should focus on usage.

Reply:
Done.